gtda/homology/simplicial.py

```diff
@@ -216,6 +218,496 @@ def transform(self, X, y=None):
                                    self.infinity_values_, self.n_jobs)
         return Xt
 
+    def plot(self, Xt, sample=0, **layout):
+        """Plot a single persistence diagram.
+
+        Parameters
+        ----------
+        ----------
+        Xt : ndarray of shape (n_samples, n_features, 3)
+            Array of persistence diagrams computed from the feature arrays or
+            distance matrices in `X`. ``n_features`` equals
+            :math:`\\sum_q n_q`, where :math:`n_q` is the maximum number of
+            topological features in dimension :math:`q` across all samples in
+            `X`.
+
+        sample : int, optional, default: ``0``
+            Index of the sample to be plotted.
+
+        layout : dict
+            Dict of string/value properties that will be passed to the
+            `plotly.graph_objs.Layout` constructor. Supported dict properties:
```

Comment:
Can we just redirect to the plotly docs? I mean, this is unreadable, and not that easy to maintain.

Reply:
Removed `**layout`